apps/web/app/setup/member/invite/page.tsx

```diff
@@ -0,0 +1,36 @@
+import { InviteMembers } from "@/app/setup/member/invite/components/InviteMembers";
+import { Metadata } from "next";
+import { getServerSession } from "next-auth";
+import { redirect } from "next/navigation";
+
+import { authOptions } from "@formbricks/lib/authOptions";
+import { SMTP_HOST, SMTP_PASSWORD, SMTP_PORT, SMTP_USER } from "@formbricks/lib/constants";
+import { getOrganizationsByUserId } from "@formbricks/lib/organization/service";
+
+export const metadata: Metadata = {
+  title: "Invite",
+  description: "Open-source Experience Management. Free & open source.",
+};
+
+const Page = async () => {
+  const session = await getServerSession(authOptions);
+
+  if (!session) {
```

Comment:
We have the checks if a user is logged in for the `/environments/..` routes in the middleware.ts
I think it makes sense that we do all the auth checks in the middleware and have a good list of routes there that needs auth instead of needing to maintain this on every single page.

Reply:
The middleware route also handles the automatic redirect. So if a user is not logged in they will get redirected to the login and afterwards to the same page again.